bin/propolis-server/src/lib/server.rs

```diff
@@ -629,15 +629,11 @@ async fn instance_serial(
             )
             .await
             {
-                error!(err_log, "Failed to spawn instance serial task: {}", e);
+                error!(err_log, "Serial task failed: {}", e);
             }
         });
 
-        super::serial::SerialTask {
-            task,
-            close_ch: Some(close_ch),
-            websocks_ch,
-        }
+        super::serial::SerialTask { task, close_ch: close_ch, websocks_ch }
```

Comment:
small, nonblocking: i think `close_ch: close_ch` can be simplified

Reply:
right you are